tests/integration/conftest.py

```diff
@@ -41,4 +41,4 @@ def get_auth_token():
 
     finetuner.login()
 
-    return finetuner.ft
+    yield finetuner.ft
```

Comment:
Should we do `return` because we don't follow up with anything?

Reply:
makes no too much difference